playbooks/rhv/README.md

```diff
@@ -0,0 +1,55 @@
+# RHV Playbooks
+## Provisioning
+This subdirectory contains the Ansible playbooks used to deploy 
+an OpenShift Container Platform environment on Red Hat Virtualization
+
+### Where do I start?
+Choose a host from which Ansible plays will be executed. This host must have
+the ability to access the web interface of the RHV cluster engine and the
+network on which the OpenShift nodes will be installed. We will refer to
+this host as the *bastion*.
```

Comment:
Why not use the engine machine for the deployment?
This will simplify the vars setting for the users.

Reply:
I disagree with directing anybody to run commands on the engine vm. The goal of simplifying for the users is better served by showing all the variables needed for the general case.

Reply:
+1

Reply:
@sradco That's not in line with the convention of the playbooks/{provider} structure in this repo.
Here, we're separating out provisioning from OpenShift install, thus the two files with vars in each that need editing.